data/tokens.json

```diff
@@ -1709,6 +1707,14 @@
             "description": "A muted version of the primary color used for the border of interactive elements that don't need to be as strong as the primary color."
```

Comment:
I suggest updating the language from primary color here to something more accurate like the default interactive color (since there's no guarantee that the interactive color will be the same as the UI's primary color over time).